Sources/MongoSwift/BSON/Document.swift

```diff
@@ -43,6 +43,13 @@ public class DocumentStorage {
 /// A struct representing the BSON document type.
 @dynamicMemberLookup
 public struct Document {
+    /// Error thrown when calls to bson_new return a null pointer.
+    internal static let InvalidBsonError = InternalError(message: "Invalid BSON data")
```

Comment:
nit: we usually prefer all caps `BSON`, even in variable names

Reply:
Done.